aspnetcore/toc.md

```diff
@@ -131,21 +131,21 @@
 ## [Logging](xref:fundamentals/logging/index)
 ### [Logging with LoggerMessage](xref:fundamentals/logging/loggermessage)
 ## [Handle errors](fundamentals/error-handling.md)
-## [File Providers](fundamentals/file-providers.md)
 ## [Host](xref:fundamentals/host/index)
 ### [Web Host](xref:fundamentals/host/web-host)
 ### [Generic Host](xref:fundamentals/host/generic-host)
 ### [Background tasks with hosted services](xref:fundamentals/host/hosted-services)
-## [Session and app state](xref:fundamentals/app-state)
 ## [Servers](xref:fundamentals/servers/index)
 ### [Kestrel](xref:fundamentals/servers/kestrel)
 ### [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module)
 ### [HTTP.sys](xref:fundamentals/servers/httpsys)
+## [Session and app state](xref:fundamentals/app-state)
+## [File Providers](fundamentals/file-providers.md)
 ## [Repository pattern](xref:fundamentals/repository-pattern)
 ## [Globalization and localization](xref:fundamentals/localization)
 ### [Configure Portable Object localization with Orchard Core](xref:fundamentals/portable-object-localization)
 ## [Initiate HTTP requests](xref:fundamentals/http-requests)
-## [Request features](xref:fundamentals/request-features)
+## [Request Features](xref:fundamentals/request-features)
```

Comment:
Is there a word that can be used instead of "Features"?

Reply:
That's their official engineering name ...

https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.http.features ... the API is *full* of the word "feature."